src/app/shared/custom-components/toolbar/toolbar.component.scss

```diff
@@ -1,7 +1,8 @@
 .toolbar {
-  background-color: coral;
+  background-color: #fff;
   display: flex;
   height: 64px;
+  align-items: center;
 
   @media screen and (max-width: 768px) {
```

Comment:
we have the variable "$tablet" in variables.scss, you can use this instead 768px (+ you will need a line 
```@import '/src/app/styles/variables.scss';```)

Reply:
done!